### ISSUE TITLE
Switch to the official `npm` module

### ISSUE THREAD

**Issue by mmomtchev** [mmomtchev/sqlite-wasm-http#226]
It seems that there is a semi-official community driven `npm` module now:

https://github.com/sqlite/sqlite-wasm

Switch to using this instead of a manual tarball download

**Comment by Row** [comment]
Is there a timeline for this?

Sorry to read about your troubles, hope you're doing ok!

**Comment by mmomtchev** [comment]
No, since I was evicted from my home as part of the extortion with the French police and the biggest companies in geospatial data, I have only a very limited access to electricity. I see that now, people like you, are not posting simultaneously anymore, but try to post a few hours apart. Alas, this does not improve my electricity situation?

**Comment by mmomtchev** [comment]
Among the companies taking part in the extortion, I would like to cite especially Camptocamp, but also Esri and Mapbox. There are also some other smaller companies, some of them with many years of experience with geospatial data.
